site/gulpfile.js

```diff
@@ -98,34 +102,34 @@ gulp.task('swag-img:optimize', () => {
 
 gulp.task('swag-img', gulp.series('swag-img:clean', 'swag-img:download' , 'swag-img:optimize'));
 
-gulp.task('clean', () => {
-    return del([
-        './rev-manifest.json',
-        'dist/assets/css/*',
-        'dist/assets/js/*',
-        'dist/assets/swag-img/*'
-    ]);
-});
+gulp.task('clean:styl', () => del('dist/assets/css/*'));
+gulp.task('clean:js', () => del('dist/assets/js/*'));
+gulp.task('clean:assets', gulp.parallel('clean:styl', 'clean:js'));
+gulp.task('clean:pug', () => del('dist/index.html'));
+gulp.task('clean:rev', () => del('dist/rev-manifest.json'));
+gulp.task('clean', gulp.parallel('clean:pug', 'clean:assets', 'clean:rev'));
 
 gulp.task('cachebust', cb => {
+    const revPath = './dist/rev-manifest.json';
+    const basePath = 'dist/assets';
     const bustedFiles = [
         'dist/assets/css/*',
         'dist/assets/js/*',
         'dist/assets/swag-img/*'
     ];
 
-    return gulp.src(bustedFiles, {base: 'dist/assets/'})
+    return gulp.src(bustedFiles, {base: basePath})
         .pipe(cachebust())
         .pipe(cacheClean())
-        .pipe(gulp.dest('dist/assets'))
-        .pipe(cachebust.manifest({
-            base: 'dist/assets',
-            merge: true
+        .pipe(gulp.dest(basePath))
+        .pipe(cachebust.manifest(revPath, {
```

Comment:
This function should support passing the path in as part of the options object

`cache.manifest({path: revPath, /*...*/})`